src/webgpu/web_platform/reftests/canvas_complex_bgra8unorm_copy_buffer_to_texture.html

```diff
@@ -12,7 +12,6 @@
   <script src="/common/reftest-wait.js"></script>
   <script type="module">
     import { run } from './canvas_complex.html.js';
-    run('bgra8unorm');
-    // TODO: make a copy of this test for rgba16float
+    run('bgra8unorm', 'copyBufferToTexture');
```

Comment:
_Option_ to consider: we could combine multiple tests into a single html by putting multiple canvases in the test (and multiple canvases in the ref). We did this in #689 - though in that test, the cases are all very similar and there wouldn't have been value in splitting them. In these tests, the different cases are meaningfully different, but we could still consider combining some (e.g. B2T+T2T+EI2T could be one, DrawTextureSample and DrawVertexColor could be another). This would at least reduce the explosion of html files when we add more formats.

Formats could be combined too - like bgra8unorm-srgb + rgba8unorm-srgb. TBH eventually I think we would put every format in every case, but today we don't support all formats so we can't do that.